src/lib/server/main_loop.c

```diff
@@ -255,9 +255,8 @@ int main_loop_init(void)
 
 #ifdef HAVE_SYSTEMD_WATCHDOG
 	if (sd_watchdog_interval.tv_sec || sd_watchdog_interval.tv_usec) {
-		struct timeval	now;
-		fr_time_to_timeval(&now, fr_event_list_time(event_list));
-		sd_watchdog_event(event_list, &now, NULL);
+		fr_time_t now = fr_time();
```

Comment:
No, this is incorrect, sd_watchdog_event takes now as a timeval, i.e. wallclock time.  This code shouldn't change.

Reply:
@arr2036 
`static void sd_watchdog_event(fr_event_list_t *our_el, fr_time_t now, void *ctx) { ... }`